ios/CovidShield/ExposureNotification.m

```diff
@@ -78,6 +76,12 @@ - (void)invalidate
       break;
     case ENStatusRestricted: resolve(@"restricted");
       break;
+    case ENStatusPaused: resolve(@"paused");
```

Comment:
We'll test this next week to see what happens but we might want to return unknown for this case.  At least until we have better info on the paused status.